cmd/init.go

```diff
@@ -159,9 +160,21 @@ func getDevelopmentEnvironment(language string) (*model.Dev, string, error) {
 }
 
 func getDeploymentName(name string) string {
-	deploymentName := filepath.Base(name)
-	deploymentName = strings.ToLower(deploymentName)
-	deploymentName = model.ValidKubeNameRegex.ReplaceAllString(deploymentName, "-")
-	log.Infof("deployment name: %s", deploymentName)
+	var deploymentName string
+	defaultDeploymentName := filepath.Base(name)
+	defaultDeploymentName = strings.ToLower(defaultDeploymentName)
+	defaultDeploymentName = model.ValidKubeNameRegex.ReplaceAllString(defaultDeploymentName, "-")
+	log.Debugf("default dployment name: %s", defaultDeploymentName)
+
+	fmt.Printf("Which deployment do you want to use for your development environment? [%s]: ", defaultDeploymentName)
```

Comment:
This is very verbose, and I dont think it's clear. I'd just go with `Deployment name [%s]:`.